pkg/types/validation/installconfig.go

```diff
@@ -1021,6 +1021,12 @@ func validateFeatureSet(c *types.InstallConfig) field.ErrorList {
 				allErrs = append(allErrs, field.Forbidden(field.NewPath("compute", "platform", "vsphere", "zones"), errMsg))
 			}
 		}
+
+		if c.Azure != nil {
```

Comment:
nit: You _could_ combine the two if statements but this looks good.

Reply:
@barbacbd This feature is marked for TechPreview for 4.13 and @jinyunma during pre-merge tests [mentioned in the findings](https://issues.redhat.com/browse/CORS-2416?focusedCommentId=21635681&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-21635681), featureSet validation is not present for this TechPreview feature. Is this check required?

Reply:
@bharath-b-rh that is correct you should add the techPreview Feature set. You _should_  add a check in `pkg/types/validation/installconfig.go` in the `validateFeatureSet` function. If its not already, you should also change the install config comments to note that this is a TechPreview feature.

Reply:
incorporated comment.